python/lsst/afw/cameraGeom/utils.py

```diff
@@ -730,7 +730,11 @@ def makeImageFromCamera(camera, detectorNameList=None, background=numpy.nan, buf
 
     boxList = getCcdInCamBBoxList(ccdList, binSize, pixelSize_o, origin)
     for det, bbox in zip(ccdList, boxList):
-        im = imageSource.getCcdImage(det, imageFactory, binSize)[0]
+        try:
+            im = imageSource.getCcdImage(det, imageFactory, binSize)[0]
+        except Exception as exc:
```

Comment:
Don't we usually write this as `Exception as e:`?  If not, no need to change it.

Reply:
This is often writen as `except Exception as e:`, but using a single-character `e` for a variable name bugs me, and I have used `exc` instead of `e` for quite a while. I believe this is a common pattern in the python world, e.g.:
* https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement
* https://www.python.org/dev/peps/pep-0008/#id51